<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="click_scienstist_male">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="click_scientist_female">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Scientist_Firstname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please Enter First Name ....</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_scientist_year_born">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>year born..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_scientist_yeardied">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>year died..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Scientist_lastname">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter last name...</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_firstname">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_lastname">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_gender">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scientist_yearborn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Born</string>
   </property>
  </widget>
  <widget class="QLabel" name="Scienstist_yeardied">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Died</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Scientist">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Scientist</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
